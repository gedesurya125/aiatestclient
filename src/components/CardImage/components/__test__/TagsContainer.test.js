import React from "react";
import reactDOM from "react-dom";
import { render, cleanup } from "@testing-library/react";
import TagContainer from "../TagsContainer";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("render without crashing", () => {
  const div = document.createElement("div");
  reactDOM.render(<TagContainer tagArray={["super", "good", "cool"]} />, div);
  reactDOM.unmountComponentAtNode(div);
});

it("render component correctly", () => {
  const { getByTestId } = render(<TagContainer tagArray={["super", "cool"]} />);
  expect(getByTestId("tags-container")).toHaveTextContent(
    'super', 'cool'
  );
});


it("to matches snapshot 1", () => {
  const tree = renderer.create(<TagContainer tagArray={["super", "cool", "okay"]}/>);
  expect(tree).toMatchSnapshot();
})