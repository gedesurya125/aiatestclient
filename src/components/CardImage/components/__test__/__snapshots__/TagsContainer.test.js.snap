// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to matches snapshot 1 1`] = `
<div
  className="css-1mk82ib"
  data-testid="tags-container"
>
  <div
    className="css-1ebnpkz"
  >
    super
  </div>
  <div
    className="css-1ebnpkz"
  >
    cool
  </div>
  <div
    className="css-1ebnpkz"
  >
    okay
  </div>
</div>
`;
